docs/resources/gke_cluster.md

```diff
@@ -0,0 +1,458 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "rafay_gke_cluster Resource - terraform-provider-rafay"
+subcategory: ""
+description: |-
+  
+---
+
+# rafay_gke_cluster (Resource)
+
+
+
+## Example Usage
+
+```terraform
+resource "rafay_gke_cluster" "tf-example" {
+  metadata {
+    name    = var.name
+    project = "defaultproject"
+  }
+  spec {
+    type = "gke"
+    blueprint {
+      name    = "minimal"
+      version = "latest"
+    }
+    cloud_credentials = "-dev-"
+    config {
+      gcp_project           = "dev-382813"
+      control_plane_version = "1.26"
+      location {
+        type = "zonal"
+        config {
+          zone = "us-central1-c"
+        }
+      }
```

Comment:
Please remove all the comments in the example. If we want to specify another example config, lets add an example in the examples directory.